chromium_src/chrome/browser/ui/views/permission_bubble/permission_prompt_bubble_view.cc

```diff
@@ -5,9 +5,22 @@
 
 #include <vector>
 
+#include "base/feature_list.h"
```

Comment:
Please, make sure the GN target has deps on all the new #includes

Reply:
thanks! added one dep.